node-cli/users.js

```diff
@@ -0,0 +1,106 @@
+const inquirer = require("inquirer");
+const Realm = require("realm");
+const index = require("./index");
+const config = require("./config");
+const main = require("./main");
+const output = require("./output");
+
+const REALM_APP_ID = config.realmAppId;
+const appConfig = {
+  id: REALM_APP_ID,
+  timeout: 10000,
```

Comment:
Out of curiosity, did you experience any issues with the default value of this? If not, perhaps leave it out to limit confusion.

Reply:
@kraenhansen default value of which?

Reply:
@MongoCaleb `timeout` is by default 1000.